examples/src/bin/sub_organization_secp256k1.rs

```diff
@@ -0,0 +1,97 @@
+use std::error::Error;
+use std::{env, vec};
+
+use turnkey_client::generated::DeleteSubOrganizationIntent;
+use turnkey_client::generated::{
+    immutable::common::v1::{AddressFormat, ApiKeyCurve, Curve, PathFormat},
+    ApiKeyParamsV2, CreateSubOrganizationIntentV7, RootUserParamsV4, WalletAccountParams,
+    WalletParams,
+};
+
+use turnkey_examples::load_api_key_from_env;
+
+use turnkey_client::TurnkeySecp256k1ApiKey;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let api_key = load_api_key_from_env()?;
+
+    // Sub-org root user gets a SECP256K1 API key:
+    let sub_organization_api_key = TurnkeySecp256k1ApiKey::generate();
```

Comment:
Let's consolidate this in the `sub_organization` example, and create 2 API keys for the root user? One secp256k1, one p256?

As-is this example won't run on CI -- we can add this to this list of course:
https://github.com/tkhq/rust-sdk/blob/c08b31f3e23a06fab7a28b49a457d6b456565f95/Makefile#L40-L45

...but I'd rather consolidate and over time have a "maximal" sort of example demonstrating/testing the different features in one go.